bindings/python/meson.build

```diff
@@ -0,0 +1,26 @@
+project(
+  'tree-sitter-capnp',
+  'c',
+  license: 'MIT'
+)
+
+python = import('python').find_installation('python3', modules: ['cffi'], pure: false)
+python_dep = python.dependency(required: true)
+
+ts_cffi_srcs = configure_file(
+  command: [python, files('ffibuilder.py'), '@OUTPUT@'],
+  output: '_tree_sitter_capnp.c',
+)
+
+ts_pyext = python.extension_module(
+    '_tree_sitter_capnp',
+    ts_cffi_srcs,
+    files('..' / '..' / 'src' / 'parser.c'),
+    # if you have a scanner, uncomment next line
+    # files('..' / '..' / 'src' / 'scanner.c'),
+    dependencies: [python_dep],
```

Comment:
Since you require 0.64.0 anyway, you can drop this explicit dependency on python.dependency(), it is automatically added for you if you don't provide one since 0.63.0.